go/cpu/bpf/dis.go

```diff
@@ -0,0 +1,161 @@
+package bpf
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/lunixbochs/usercorn/go/models"
+)
+
+type ins struct {
+	addr  uint64
+	op    uint16
+	jt    uint8
+	jf    uint8
+	k     uint32 // Depends on addressing mode?
```

Comment:
why store as 32-bit if values can be 64-bit in other addressing modes? mem and regs backends will enforce sizes for you at runtime